custom-vu/src/main/kotlin/jces1209/vu/page/DcIssuePage.kt

```diff
@@ -57,4 +60,53 @@ class DcIssuePage(
     override fun linkIssue(): DcIssueLinking {
         return DcIssueLinking(driver)
     }
+
+    override fun changeAssignee(): DcIssuePage {
+        var assignee = "Tdztbchh Eskfsyas"
+
+        driver
+            .findElement(By.cssSelector("[id = 'assignee-val']"))
+            .click();
+
+        val userPiccker = driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"assignee-form\"]"))
+            )
+
+        Actions(driver)
+            .sendKeys(Keys.DELETE, Keys.ARROW_DOWN)
+            .perform()
+
+        driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"assignee-group-suggested\"]"))
+            )
+
+        var selectedUser = driver
+            .wait(
+                ExpectedConditions.elementToBeClickable(By.xpath("//*[@id=\"all-users\"]/li[3]/a"))
```

Comment:
`li[3]` - looking for the third list item.
It's not semantic, it's very tightly bound to the shape of the DOM, which changes more often than semantics. It'll cost more maintenance. I'd try to find a more semantic selector, based on names/content rather than position.

- [ ] try to avoid overly specific selectors